src/argilla/tasks/users/create_default.py

```diff
@@ -30,17 +31,17 @@ def create_default(quiet: bool):
                 first_name="",
                 username=DEFAULT_USERNAME,
                 role=UserRole.admin,
-                api_key=DEFAULT_API_KEY,
-                password_hash=accounts.hash_password(DEFAULT_PASSWORD),
+                api_key=settings.api_key,
```

Comment:
If these settings are used only for the argilla task, I would prefer using a click.option instead, and calling the tasks:

```bash
python -m argilla.tasks.users.create_default --password $MY_PASS --api-key $MY_API_KEY
```

For me, it's a bit weird having app env vars used only on this task.

Reply:
I Agree. The only difference is users can create a default user without looking at the docs with `users.create_default` but not if they use `users.create`

Remember that the `users.create_default` is a shortcut for a quick start installation without docker.